### PR TITLE
Add QUARTZ token to prices.usd table

### PR DESCRIPTION
**Pull request by fyang1024**
### Contribution type
Please check the type of contribution this pull request is for:

- [ ] New spell(s)
- [x] Adding to existing spell lineage
- [ ] Bug fix

---

### For adding to existing spell lineage
If you are adding to an existing spell lineage, please provide the following information:

- **Description:** Added SandClock (sandclock.org) QUARTZ token to the list of ethereum tokens.

### CODE REVIEW

**Comment by fyang1024** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @fyang1024

File: models/prices/ethereum/prices_ethereum_tokens.sql [hunk 1]
```diff
@@ -1793,7 +1793,8 @@ FROM
     ('bigtime-big-time', 'ethereum', 'BIGTIME', 0x64Bc2cA1Be492bE7185FAA2c8835d9b824c8a194, 18),
     ('portal2-portal-coin', 'ethereum', 'PORTAL', 0x1bbe973bef3a977fc51cbed703e8ffdefe001fed, 18),
     ('dough-l3t-h1m-c00k', 'ethereum', 'DOUGH', 0xFc116eA24F002F600e363bdce4b91715fe5e0392, 9),
-    ('giv-giv', 'ethereum', 'GIV', 0x900db999074d9277c5da2a43f252d74366230da0, 18)
+    ('giv-giv', 'ethereum', 'GIV', 0x900db999074d9277c5da2a43f252d74366230da0, 18),
+    ('quartz-sandclock', 'ethereum', 'QUARTZ', 0xba8a621b4a54e61c442f5ec623687e2a942225ef, 18)
```
Comment (jeff-dude) [inline models/prices/ethereum/prices_ethereum_tokens.sql:1797]: according to the CI test result, it appears this token is not active? we won't get any new pricing info, if merged

Comment (fyang1024) [reply models/prices/ethereum/prices_ethereum_tokens.sql:1798]: I merged the latest main, now all the tests have passed. I think the error was quite occasional

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**